app/src/main/java/fr/free/nrw/commons/CommonsApplication.java

```diff
@@ -339,4 +345,47 @@ public interface LogoutListener {
 
         void onLogoutComplete();
     }
+
+    public static class BaseLogoutListener implements CommonsApplication.LogoutListener {
```

Comment:
Kindly add some JavaDocs for this listener class which helps others understand its purpose and what it does and the kinds of invocation that it supports.